draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -753,6 +753,80 @@ any stream which references the failing certificate in question or process the
 requests as unauthenticated and provide error information at the HTTP semantic
 layer.
 
+# Certificate Extensions {#extension}
+
+## Allowed Primary Domain Certificate Extension {#extension-primary}
+
+The Allowed Primary Domain extension allows certificates to limit their use with
+Secondary Certificate Authentication.  The identities listed in this extension
```

Comment:
This doesn't say *how* the use is limited.

I assume that this means that the certificate won't be accepted as a secondary certificate unless the certificate offered in the TLS handshake contains one of the identified names  in subjectAltName?

The name of the extension implies that there is only one, but this appears to allow many-to-many matching, which might be a bit gross to implement.

Reply:
Yes, that restriction is down in the Security Considerations.  I can add a ref, unless you think it belongs here (or elsewhere).